spec/config/initializers/filter_parameter_logging_spec.rb

```diff
@@ -7,6 +7,9 @@
   let(:parameter_filter) { ActiveSupport::ParameterFilter.new(config.filter_parameters) }
 
   it 'filters query from logs' do
-    expect(config.filter_parameters).to match(array_including(:query))
+    filter_contains_query = config.filter_parameters.any? do |param|
```

Comment:
Why is this change necessary?  This seems to pass fine as is locally without this added complication.

Reply:
@lorawoodford  - I definitely am an agreement with you and have found that to be the case for me as well, locally ... but when running CI pipeline builds (with no change) it specifically fails on this spec.

Reply:
I ran it a few times with variations on the test implementation and the error message changes accordingly with it as well. This implementation did the trick and passed.

Reply:
Ok, what about something like:

```
expect(config.filter_parameters.to_s).to match(/:query\b/)
```
That should match both locally when we get back:
```
[:password, :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn, :query]
```
and in CI when we (for whatever reason) get back:
```
[/(?i:password)|(?i:passw)|(?i:secret)|(?i:token)|(?i:_key)|(?i:crypt)|(?i:salt)|(?i:certificate)|(?i:otp)|(?i:ssn)|(?i:query)/]
```
I'm still a little curious/concerned why we're getting back different things in CI and locally... when we smoke test we'll definitely want to ensure this is still behaving as described in SRCH-3933.